docs/_static/auth/iam.yaml

```diff
@@ -136,6 +136,39 @@ paths:
                 "$ref": "#/components/schemas/OAuthAuthorizationServerMetadata"
         default:
           $ref: '../common/error_response.yaml'
+#  TODO: what should this path be? It is not a /.well-known/. It will be provided in an authorization request as the "client_metadata_uri"
```

Comment:
will be temporary right?

Reply:
what do you mean? OpenID4VP declares `client_metadata` and `client_metadata_uri` parameters for the authorization request. Since the amount of characters in the metadata is too much to add to the query params it should be provided as `client_metadata_uri`. Having to request the client metadata at the client itself also prevents tampering with the `redirect_uris`. So I don't think this is temporary

Reply:
right, but it's defined at runtime. So we can decide whatever we want.

Reply:
Ahh.., then yes. I will remove the comment